cms/djangoapps/contentstore/views/course.py

```diff
@@ -1038,6 +1042,9 @@ def rerun_course(user, source_course_key, org, number, run, fields, background=T
     fields['enrollment_end'] = None
     fields['video_upload_pipeline'] = {}
 
+    # Enable certain fields rolling forward, where configured
+    fields['force_on_flexible_peer_openassessments'] = enable_flexible_peer_openassessments_on_rerun(source_course_key)
+
```

Comment:
If the previous run had turned the course setting on but the org has the wafffle off, this would force the setting off. I don't think that's what we want.

```suggestion
    if enable_flexible_peer_openassessments_on_rerun(source_course_key):
    fields['force_on_flexible_peer_openassessments'] = True 
```

Reply:
Ah, so basically only rolling-forward the feature, never toggling it off? I could buy that.

Reply:
@jansenk , this should be updated, welcoming re-review